src/views/GlossaryPage.vue

```diff
@@ -0,0 +1,279 @@
+<template>
+  <DashboardLayout>
+    <DashboardCard>
+      <template #title> Agencies and Organizations </template>
+      <template #content>
+        <div class="px-4" style="font-size: 0.95rem">
```

Comment:
inline styles are generally an anti-pattern (especially if not using logic to determine them).  I'd also stick to the [bulma font sizes](https://bulma.io/documentation/helpers/typography-helpers/) so we use a consistent set of design tokens throughout the app